go/hyperkit.go

```diff
@@ -573,11 +573,16 @@ func (h *HyperKit) execHyperKit() error {
 	if err != nil {
 		log.Debugf("hyperkit: Cannot write state: %v, killing %v", err, h.Pid)
 		h.process.Kill()
-		return err
+		return nil, err
 	}
+	return cmd, nil
+}
+
+// execHyperKit waits for hyperkit, in background if demanded.
+func (h *HyperKit) execHyperKit(cmd *exec.Cmd) error {
```

Comment:
exec is now wrong for this function. How about `waitHyperkit` or even just `wait` (given it is invoked as `h.wait()`)?

Reply:
This one is dead and buried by the final commit as well, oops!